iml-agent/src/action_plugins/high_availability/mod.rs

```diff
@@ -85,6 +87,105 @@ where
     Ok(String::from_utf8_lossy(&o.stdout).to_string())
 }
 
+pub(crate) async fn pcs(args: Vec<String>) -> Result<String, ImlAgentError> {
+    let o = Command::new("pcs").args(args).checked_output().await?;
+
+    Ok(String::from_utf8_lossy(&o.stdout).to_string())
+}
+
+async fn crm_mon_status() -> Result<Element, ImlAgentError> {
+    let o = Command::new("crm_mon")
+        .args(&["--as-xml", "--inactive"])
+        .checked_output()
+        .await?;
+
+    Ok(Element::from_reader(o.stdout.as_slice())?)
+}
+
+fn resource_status(tree: Element) -> HashMap<String, String> {
+    let resources = tree.find("resources").unwrap();
```

Comment:
Nit: this is similar to what I'm doing in terms of reading crm_mon resources. We should consolidate our changes.